vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -14,9 +14,6 @@
 {% block current_title %}{{ dataset.title }}{% endblock %}
 
 {% block head %}
-    <link href="{% static '/css/datatables.min.css' %}" rel="stylesheet">
-    <script src="{% static '/js/jquery-3.6.0.min.js' %}"></script>
-    <script src="{% static '/js/datatables.min.js' %}"></script>
     <script src="{% static 'hitcount/jquery.postcsrf.js' %}"></script>
```

Comment:
This should be included into webpack. In order to do that, first you need to collect static files:

    poetry run python manage.py collectstatic

An then add directory where static files are collected into [resolve.modules](https://webpack.js.org/configuration/resolve/#resolvemodules) and then import this this file as webpack module. And do the same for other places where postcsrf.js is used.

Currently `collectstatic` collects static files into `static/` directory, this should go into `var/static/`. You can change that in settings `STATIC_ROOT`.